org.osgi.test.junit5/test.bndrun

```diff
@@ -44,17 +44,17 @@
     begin=-1
 -runbundles: \
 	assertj-core;version='[3.19.0,3.19.1)',\
-	junit-jupiter-api;version='[5.7.1,5.7.2)',\
-	junit-jupiter-engine;version='[5.7.1,5.7.2)',\
-	junit-jupiter-params;version='[5.7.1,5.7.2)',\
-	junit-platform-commons;version='[1.7.1,1.7.2)',\
-	junit-platform-engine;version='[1.7.1,1.7.2)',\
-	junit-platform-launcher;version='[1.7.1,1.7.2)',\
-	junit-platform-testkit;version='[1.7.1,1.7.2)',\
-	net.bytebuddy.byte-buddy;version='[1.10.20,1.10.21)',\
-	net.bytebuddy.byte-buddy-agent;version='[1.10.20,1.10.21)',\
-	org.mockito.mockito-core;version='[3.8.0,3.8.1)',\
-	org.objenesis;version='[3.1.0,3.1.1)',\
+	junit-jupiter-api;version='[5.7.2,5.7.3)',\
+	junit-jupiter-engine;version='[5.7.2,5.7.3)',\
+	junit-jupiter-params;version='[5.7.2,5.7.3)',\
+	junit-platform-commons;version='[1.7.2,1.7.3)',\
+	junit-platform-engine;version='[1.7.2,1.7.3)',\
+	junit-platform-launcher;version='[1.7.2,1.7.3)',\
+	junit-platform-testkit;version='[1.7.2,1.7.3)',\
+	net.bytebuddy.byte-buddy;version='[1.11.0,1.11.1)',\
+	net.bytebuddy.byte-buddy-agent;version='[1.11.0,1.11.1)',\
+	org.mockito.mockito-core;version='[3.10.0,3.10.1)',\
+	org.objenesis;version='[3.2.0,3.2.1)',\
```

Comment:
I recall that our strategy was generally to keep these on lower versions to verify that our code doesn't always require the latest and greatest; however we were happy to bump the version if there was an established need. I was wondering if there  was a particular need that you had established or else this was just a general catch-up?

Reply:
I have been merging the dependabot PRs which don't handle these file. This is just catching up with the values in the pom.